calico/acl_manager/acl_manager.py

```diff
@@ -37,6 +37,7 @@ def main():
     config.read(args.config_file or 'acl_manager.cfg')
     plugin_address = config.get('global', 'PluginAddress')
     log_file_path = config.get('log', 'LogFilePath')
+    bind_address = config.get('global', 'BindAddr')
```

Comment:
Maybe. My rationale for "BindAddr" is that this is the address that the process binds to - which is also a local address.

I think I don't really care, and it is trivial to change. Do you still want me to change it, or have a convinced you that you don't care either?